app/graphql/mutations/integration_items/fetch_tax_items.rb

```diff
@@ -20,6 +20,8 @@ def resolve(**args)
 
         ::Integrations::Aggregator::SyncService.call(integration:)
 
+        integration.integration_items.where(item_type: :tax).destroy_all
```

Comment:
Same question, as it feels weird to have "business logic" in the mutation

Reply:
Moved it to the items service